adaptors/extensions.js

```diff
@@ -1,59 +1,63 @@
-var fs = require('fs');
+"use strict";
 
-exports.clearTemporaryFile = function clearTemporaryFile(req, reply) {
-  if (req.response.isBoom) {
+const fs = require(`fs`);
+
+exports.clearTemporaryFile = function(request, reply) {
+  if (request.response.isBoom) {
     return reply.continue();
   }
 
   // Once a successful request completes, we delete any
   // temporary files we created
-  req.response.once('finish', function() {
-    if (!req.app.tmpFile) {
+  request.response.once(`finish`, function() {
+    if (!request.app.tmpFile) {
       return;
     }
 
-    fs.unlink(req.app.tmpFile, function(err) {
+    fs.unlink(request.app.tmpFile, function(err) {
       if (err) {
-        req.log.error('Failed to destroy temporary file with ' + err);
+        request.log.error(`Failed to destroy temporary file with ${err}`);
       }
     });
   });
 
   reply.continue();
 };
 
-exports.logRequest = function logRequest(req, reply) {
+exports.logRequest = function(request, reply) {
+  const response = request.response;
+
   // We don't want to clutter the terminal, so only
   // show request details if this was an error
-  if (!req.response.isBoom) {
+  if (!response.isBoom) {
     reply.continue();
     return;
   }
 
-  var data = req.response.data;
-  var error = data && data.error;
+  const data = response.data;
+  const error = data && data.error;
 
   // Prefer the error object stack over the
   // boom object stack
-  var stack = error && error.stack || req.response.stack;
+  const stack = error && error.stack || response.stack;
+
+  let logLevel = `error`;
```

Comment:
there shouldn't be a "choice", their function is clearly defined in the spec. If you need a function level variable, you use `var`, if you need a block scoped variable, you use `let`, so that should be the guiding principle: does it need function scoping? use `var`. Does it need to only exist for as long as the current block is active? use `let`.